experiments/integrated/fluxnet/fluxnet_domain.jl

```diff
@@ -8,7 +8,3 @@ h_canopy = h_stem + h_leaf
 compartment_midpoints =
     n_stem > 0 ? [h_stem / 2, h_stem + h_leaf / 2] : [h_leaf / 2]
 compartment_surfaces = n_stem > 0 ? [zmax, h_stem, h_canopy] : [zmax, h_leaf]
-
-land_domain =
```

Comment:
I moved this into the driver scripts so that we have access to lat and long when we construct it. This file will be removed soon in a different PR, so it doesn't matter that this is duplicating some code.